app/scripts/modules/core/src/pipeline/executions/executionGroup/ExecutionGroup.tsx

```diff
@@ -166,6 +167,11 @@ export class ExecutionGroup extends React.Component<IExecutionGroupProps, IExecu
     e.stopPropagation();
   }
 
+  private rerunExecutionClicked(execution: IExecution): void {
+    ReactGA.event({ category: 'Pipeline', action: 'Rerun pipeline button clicked', label: this.props.group.heading });
```

Comment:
is this GA event (or the other one inside `Execution`) redundant?

Reply:
Yes, good catch. I will remove the other one.